tensorflow_io/bigtable/BUILD

```diff
@@ -53,6 +71,12 @@ cc_binary(
     srcs = [
         "kernels/test_kernels/bigtable_test_client_test.cc",
     ],
+    copts = [
+        "-pthread",
+        "-std=c++11",
+        "-D_GLIBCXX_USE_CXX11_ABI=0",
```

Comment:
@henrytansetiawan This line ` "-D_GLIBCXX_USE_CXX11_ABI=0",` could be removed as it is processed by ./configure.sh automatically now.